lava-v2-jobs-from-api.py

```diff
@@ -195,6 +202,7 @@ def get_job_params(config, template, opts, device, build, defconfig, plan):
         'nfsrootfs_url': nfsrootfs_url,
         'lab_name': config.get('lab'),
         'context': device.get('context'),
+        'rootfs_prompt': rootfs_prompt
```

Comment:
Nitpick: It's a bit nicer to have a `,` at the end of each line, so when adding more lines in the future this one won't have to be edited.